scoring-periods/1.md

```diff
@@ -0,0 +1,27 @@
+## `Scoring Period 1`
+### _DD/MM/YY - DD/MM/YY_
+<br>
+
+_Note: [Scoring Period #0](/0.md) is running at the same time, to recognize existing contributors to the project._
+<br>
+#### Announcement
+We are very happy to announce the inception of our Founding Member program. This initial scoring period will help us to calibrate the program rules and ensure the system works effectively before marketing the program more widely during `Scoring Period #2`.
+
+Existing contributors to the project should also be aware that during this first scoring period, previous contributions to the project can be reported (`Scoring Period #0`). After `Scoring Period #1` has ended, this will no longer be possible.
+
+We also wanted to recognize our Initial Founding Members, namely:
+- `@username`
+- `@username`
+- `@username`
+- `@username`
+- `@username`
+- `@username`
+
+We would like to thank these community members for their generous ongoing contributions to the project and hope that they will continue to be excellent ambassadors for Joystream as official Founding Members.
+
+
+#### Processing Note
+
+_Once we have processed all of the summaries submitted in this period and inducted any new founding members etc., a processing note will be added here with comments and some suggestions for the next scoring period._
+
+_Check back here on `DD/MM/YY` when we expect to have processed all of the qualifying activity summaries._
```

Comment:
Remember that people won't be able to submit before the end...

Reply:
Oh, I thought they could submit at all times?
Didn't we agree that would make it easier to feel an immediate sense of achievement if summaries could be submitted whenever and processed as soon as possible?
The logistics of that do seem more complex though.